js/src/get-started-page/case-studies-card/index.scss

```diff
@@ -0,0 +1,53 @@
+.gla-get-started-case-studies-card {
+	// Adjust <CardHeader> imported from @wordpress/components.
+	// Repeat selector to make it higher priority.
+	.components-card__header.components-card__header {
+		flex-direction: column;
+		padding: $grid-unit * 5.75 $grid-unit * 14 $grid-unit-50;
+		text-align: center;
+
+		@media (max-width: $break-small) {
+			padding: $grid-unit-30;
+		}
+	}
```

Comment:
💅 / 🚧 

Inserting this selector inside `.components-flex {` also works and we can avoid that double selector patch.

```
.components-flex {
		gap: $grid-unit * 3.5;
		padding: 0 $grid-unit * 7.75 $grid-unit * 5.75;

		&.components-card__header {
			flex-direction: column;
			... 
```

Reply:
Thanks for it! Using this form is clearer, updated in c673732. Also updated for other cards in fd58db1.

Note that it works for `CardHeader` since it has `.components-flex` class, while `CardBody` doesn't have that class so in `js/src/get-started-page/benefits-card/index.scss` I still use the repeating selectors hack in order to override the `CardBody`'s original styles.